packages/tldraw/src/lib/ui/hooks/useTools.tsx

```diff
@@ -57,7 +57,7 @@ export function ToolsProvider({ overrides, children }: TLUiToolsProviderProps) {
 				kbd: 'v',
 				readonlyOk: true,
 				onSelect(source) {
-					editor.setCurrentTool('select')
+					editor.setCurrentTool('select.idle')
```

Comment:
be explicit about going to the select state rather than the editing state, because editing is a child of select